prototype/Makefile

```diff
@@ -33,9 +33,15 @@ build_and_watch:
 client_start:
 	docker-compose -f build/deployments/docker-compose.yaml up -d client
 
+client_alpha_start:
```

Comment:
What is `client_alpha`? Why can't we just use one client?

Reply:
The client that will use the p2p module. I'd like to keep them both to allow easy switching off and on.